### DISCUSSION TITLE
Meaningful maxLength Attributes

### DISCUSSION THREAD

**Discussion by ghost** [OP-TED/eForms-SDK#455] [Q&A] [answered]
A lot of the textfields have a maxLength 400, which does not always make sense. e.g. the posl code will never be longer the 10 or 20 characters. When generating the UI using the SDK, this results in much too long and confusing input controls.

Is it possible to get meaningful "maxLength" attributes for the eforms fields?

**Comment by rouschr** [comment, marked as answer]
Hello, 

we do not want to be overly strict as there could be special cases in some countries.
It is possible that we could review some of the values in the future, but currently this is not a priority.
If it is maxLength 400 and you want less you can restrict to a lower value in your own application.
